src/ImageList.js

```diff
@@ -0,0 +1,21 @@
+import React, { Component } from 'react'
+import creatures from './data.js';
+import ImageItem from './ImageItem.js';
+
+export default class ImageList extends Component {
+    render() {
+        return (
+            <body>
+                {creatures.map(element =>{
```

Comment:
great work implementing the .map and passing down those props individually. Great practice to organize your info